.github/workflows/upstream-build.yml

```diff
@@ -36,58 +36,9 @@ jobs:
         run: |
           west blobs fetch hal_silabs
 
-      - name: Build Entropy test
-        working-directory: zephyr-silabs
-        shell: bash
-        run: |
-          west twister -v --inline-logs -K \
-            -s drivers.entropy \
-            -p siwx917_rb4338a
-
-      - name: Build Flash test
-        working-directory: zephyr-silabs
-        shell: bash
-        run: |
-          west twister -v --inline-logs \
-            -s drivers.flash.common.default \
-            -p siwx917_rb4338a \
-            -x DTC_OVERLAY_FILE=$(pwd)/tests/drivers/flash/common/boards/siwx917_rb4338a.overlay
-
-      - name: Build DMA test
-        working-directory: zephyr-silabs
-        shell: bash
-        run: |
-          west twister -v --inline-logs \
-            -s drivers.dma.chan_blen_transfer \
-            -p siwx917_rb4338a \
-            -x DTC_OVERLAY_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.overlay \
-            -x EXTRA_CONF_FILE=$(pwd)/tests/drivers/dma/chan_blen_transfer/boards/siwx917_rb4338a.conf
-
-      - name: Build Bluetooth samples
-        working-directory: zephyr-silabs
-        shell: bash
-        run: |
-          west twister -v --inline-logs -K \
-            -s sample.bluetooth.peripheral_hr \
-            -s sample.bluetooth.observer \
-            -p xg27_dk2602a \
-            -p siwx917_rb4338a
```

Comment:
Wouldn't we still want to keep some amount of 917 testing here, just like we do for EFR32?

Reply:
Probably yes.